app/mailers/response_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class ResponseMailer < ActionMailer::Base
+  domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
+  default from: "noreply@#{domain}"
```

Comment:
Voeg een `ApplicationMailer` basisklasse toe met configuratie die voor alle mailers hetzelfde is